### PR TITLE
Create promise

### PR DESCRIPTION
**Pull request by Ragnar-Hardarson**
This posts a promise to the backend.

This PR is super simple and could be approved upon, but I'd like to merge this minimum version as is.

Notable things missing from this PR, that should be added later:
- Test that form submits. Enzyme and MUI don't play along [nicely](https://github.com/airbnb/enzyme/issues/252)
- Better user feedback. Error message, success message, redirect
- Get list of Parliaments from Backend, currently uses a fixed ID that points to test Parliament

### CODE REVIEW

**Comment by Ragnar-Hardarson** [comment]
Note that you need to be logged in to create Promises, so make sure you hit that Login page 😄

File: .eslintrc [hunk 1]
```diff
@@ -149,7 +149,6 @@
     "constructor-super": 1,
     "no-const-assign": 2,
     "no-dupe-class-members": 2,
-    "no-duplicate-imports": 2,
```
Comment (AriHrannar) [inline .eslintrc]: Why remove this one?

File: src/utils/api.js [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+  name: string,
+  shortDescription: string,
+  longDescription: string,
+  parliamentId: number,
+};
 
 export const checkStatus = (response: Response) => {
-  if (response.status >= 200 && response.status < 300) { //eslint-disable-line
+  /* eslint-disable no-magic-numbers */
+  if (response.status >= 200 && response.status < 300) {
```
Comment (AriHrannar) [inline src/utils/api.js]: Is there some sort of javascript library that allow us to reference different status codes without having magic numbers? Django and Dropwizard (java) have something like "Response.Status.OK" for 200 for example. Might make things more readable for the future? (Not suggesting a change but might be nice?)

File: .eslintrc (hunk 1 above)
Comment (Ragnar-Hardarson) [inline .eslintrc]: Fails with Type imports. There are going to be a lot of those in the future

File: src/utils/api.js (hunk 2 above)
Comment (Ragnar-Hardarson) [inline src/utils/api.js]: I suppose https://github.com/prettymuchbryce/node-http-status covers this.

I think this particular case should have the numbers though, to really drive home the point that it's checking the entire 2XX success range.

I would get more confused if I read
```
if (response.status >= HttpStatus.OK && response.status < HttpStatus.MULTIPLE_CHOICES) {
...
```

Comment (AriHrannar) [inline src/utils/api.js]: Yes more confusing indeed

**Comment by Jarngreipr** [comment]
promise

It's ambiguous whether the Name field refers to the name of a promise or a politician. 
Should promises even have names?

File: .eslintrc (hunk 1 above)
Comment (Ragnar-Hardarson) [inline .eslintrc]: Is that ok with you @AriHrannar

**Comment by Ragnar-Hardarson** [comment]
@Jarngreipr How about Promise Title?

It's something to display in the list

**Comment by Jarngreipr** [comment]
@Ragnar-Hardarson A promise needs to be connected to either a politician or party. There is no accountability if the promise stands on its own.

**Comment by Ragnar-Hardarson** [comment]
@Jarngreipr I'm trusting you for the business rules. Don't know if they're written down somewhere.

I'll do politician specific promises for the time being

**Comment by Ragnar-Hardarson** [comment]
I added Politicians and Parties as possible mappings.

Please review again

**Comment by Ragnar-Hardarson** [comment]
Best enjoyed with https://github.com/veritus/veritus-backend/pull/38

**Comment by Ragnar-Hardarson** [comment]
> Doesnt work with the new changes.
😔  I'll fix it

> Might also be nice to include the code that wrote
Sorry could you elaborate?

> That can either be in this PR or another one if you like :)
Another PR for sure

**Comment by Ragnar-Hardarson** [comment]
@AriHrannar @Jarngreipr fixed with https://github.com/veritus/veritus-web/pull/23/commits/d178dad16ff81330991508fca7ab1fcbd45b6b6f

**Comment by AriHrannar** [comment]
Looks good, works now :) 

> Might also be nice to include the code that wrote

I wrote some code that fetches the political parties and lists then, just might be nice to include the real ones in create a promise instead of the testing ones. But that should be in a new PR

**Pull request merged**